config/Config-build.in

```diff
@@ -7,6 +7,13 @@
 
 menu "Global build settings"
 
+	config JSON_CREATE_IMAGE_INFO
+		bool "Create JSON info files per build image"
+		default y
```

Comment:
default y?

Reply:
I think this file is very handy, however surely can disable it per default

Reply:
"The option of JSON creatin is disabled by default, but active for buildbots."

That's what you said in the first message here :-)

Maybe change to "default BUILDBOT" or similar. I do not object towards having "default y", I just wanted to point out the "inconsistency" ... ;-)

Reply:
Sorry the GitHub description was outdated!